packages/react-components/react-link/library/src/components/Link/useLink.ts

```diff
@@ -14,6 +15,10 @@ export const useLink_unstable = (
   props: LinkProps,
   ref: React.Ref<HTMLAnchorElement | HTMLButtonElement | HTMLSpanElement>,
 ): LinkState => {
+  for (const token of Object.keys(tokens)) {
```

Comment:
I'd be curious if you just had static references if this changes. As of now it pulls it all in because it's all being iterated through dynamically in runtime.